src/pages/contact-us/_offices.js

```diff
@@ -160,7 +165,8 @@ const FullBox = styled(Flex)`
     }
 `
 
-export const Offices = () => {
+// TODO: remove enableBelarus boolean once address is confirmed.
+export const Offices = ({ enableBelarus }) => {
```

Comment:
`enableBelarus` should be in `snake_case`

Reply:
as a suggestion you may rename it to `is_belarus_enables` to indicate it holds a boolean value